### PR TITLE
Add ignore_warnings to README

### PR DESCRIPTION
**Pull request by blimmer**
As requested in #144. I also submitted a PR to the CodeClimate doc site.

/cc @dblandin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 2. Run `codeclimate engines:enable eslint`. This command both installs the engine and enables it in your `.codeclimate.yml` file.
 3. You're ready to analyze! Browse into your project's folder and run `codeclimate analyze`.
 
+### Configuration
+
+#### `ignore_warnings`
+By default, ESLint rules at the warning level will be registered as errors. You can ignore warnings by setting this flag:
```
Comment (dblandin) [inline README.md]: To match Code Climate parlance a bit more, I might go with something like:

"By default, this engine will emit both ESLint errors and warnings as Code Climate issues. If you prefer, you can ignore warning-level violations by setting the `ignore_warnings` configuration option:"

What do you think?

Comment (blimmer) [inline README.md]: Works for me - I'll update.

**Comment by dblandin** [comment]
Nice, thanks!

**Comment by dblandin** [comment]
ICYMI: Your contributions were featured in [a recent Code Climate Changelog post](https://codeclimate.com/changelog/581a6253e115ac0065005366)!

Thanks again! :tada:

**Comment by blimmer** [comment]
Nice - thanks for sharing @dblandin . As my team grows, CodeClimate is going to be an essential part of our toolbox. Happy to help.

**Pull request merged**